<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="55384" started-at="2017-02-04T12:51:30Z" finished-at="2017-02-04T12:52:26Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="55384" started-at="2017-02-04T12:51:30Z" finished-at="2017-02-04T12:52:26Z">
      <class name="testcases.TC05_AddBatches">
        <test-method status="PASS" signature="beforeSuite()[pri:0, instance:testcases.TC05_AddBatches@e2abd]" name="beforeSuite" is-config="true" duration-ms="598" started-at="2017-02-04T18:21:30Z" finished-at="2017-02-04T18:21:30Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeSuite -->
        <test-method status="PASS" signature="beforeTest()[pri:0, instance:testcases.TC05_AddBatches@e2abd]" name="beforeTest" is-config="true" duration-ms="1" started-at="2017-02-04T18:21:30Z" finished-at="2017-02-04T18:21:30Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeTest -->
        <test-method status="PASS" signature="setValues()[pri:0, instance:testcases.TC05_AddBatches@e2abd]" name="setValues" is-config="true" duration-ms="0" started-at="2017-02-04T18:21:30Z" finished-at="2017-02-04T18:21:30Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setValues -->
        <test-method status="PASS" signature="beforeMethod()[pri:0, instance:testcases.TC05_AddBatches@e2abd]" name="beforeMethod" is-config="true" duration-ms="1" started-at="2017-02-04T18:21:31Z" finished-at="2017-02-04T18:21:31Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method status="FAIL" signature="login(java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:testcases.TC05_AddBatches@e2abd]" name="login" duration-ms="54703" started-at="2017-02-04T18:21:31Z" data-provider="fetchData" finished-at="2017-02-04T18:22:26Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[Raj]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[6.0]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[sakthivel@testleaf.com]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[Leaf@123]]>
              </value>
            </param>
            <param index="4">
              <value>
                <![CDATA[Appium]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.RuntimeException">
            <message>
              <![CDATA[FAILED]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.RuntimeException: FAILED
	at utils.Reporter.reportStep(Reporter.java:29)
	at wrapper.WrapperMethods.clickXpathUsingActions(WrapperMethods.java:682)
	at leafOrg.BatchesPage.clickCompany(BatchesPage.java:28)
	at testcases.TC05_AddBatches.login(TC05_AddBatches.java:32)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:124)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:773)
	at org.testng.TestRunner.run(TestRunner.java:623)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1185)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1110)
	at org.testng.TestNG.run(TestNG.java:1018)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:112)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:205)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:176)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.RuntimeException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- login -->
        <test-method status="PASS" signature="afterMethod()[pri:0, instance:testcases.TC05_AddBatches@e2abd]" name="afterMethod" is-config="true" duration-ms="0" started-at="2017-02-04T18:22:26Z" finished-at="2017-02-04T18:22:26Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method status="PASS" signature="afterTest()[pri:0, instance:testcases.TC05_AddBatches@e2abd]" name="afterTest" is-config="true" duration-ms="0" started-at="2017-02-04T18:22:26Z" finished-at="2017-02-04T18:22:26Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterTest -->
        <test-method status="PASS" signature="afterSuite()[pri:0, instance:testcases.TC05_AddBatches@e2abd]" name="afterSuite" is-config="true" duration-ms="371" started-at="2017-02-04T18:22:26Z" finished-at="2017-02-04T18:22:26Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterSuite -->
      </class> <!-- testcases.TC05_AddBatches -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
